{{cookiecutter.repostory_name}}/noxfile.py

```diff
@@ -17,11 +19,15 @@
 
 nox.options.default_venv_backend = 'venv'
 nox.options.stop_on_first_error = True
-nox.options.reuse_existing_virtualenvs = True
+nox.options.reuse_existing_virtualenvs = not CI
+
 
-# In CI, use Python interpreter provided by GitHub Actions
-if CI:
-    nox.options.force_venv_backend = 'none'
+def install(session: nox.Session, *args):
+    groups = []
+    for group in args:
+        groups.extend(['--group', group])
+    session.run('pdm', 'lock', '--check', external=True)
+    session.run('pdm', 'sync', *groups, external=True)
```

Comment:
shouldn't this be just
```suggestion
    session.run('pdm', 'install', '--check', *groups, external=True)
```
?